src/views/common/common.js

```diff
@@ -1,22 +1,21 @@
 const Footer = function() {
     return `
-    <footer id='fixed-footer'>
-    <div class="container">
-        <p class="footer-paragraph">Copyright@Softvision 2017</p>
-        </div>
+        <footer id='footer'>
+            <div class="container">
+                <p class="footer-paragraph"> Copyright@Softvision 2017 </p>
+            </div>
         </footer>
     `;
 
 }
 const NAV = function(options={}) {
     return `
-    <section class="topnav" id="Topnav">
-    <img src="assets/logo.png" style="width:15%; height:30px;float:left">
-    <a href="#evaluations">Evaluations</a>
-    <a href="#new evaluation">New Evaluations</a>
-    <a style="float:right" href="#about">Logout</a>
-    <hr class="eval-header-bottom eval-header-bottom-skin">
-  </section>
+        <section class="topnav" id="Topnav">
+            <img src="assets/logo.png" style="width:15%; height:30px;float:left">
+            <a href="#evaluations">Evaluations</a>
+            <a href="#new evaluation">New Evaluations</a>
```

Comment:
Putting spaces in `href` / `src` attributes might or might not work as expected, depending on browser / app. The safest option would be to safely encode empty spaces as `%20` to make sure everybody behaves consistently and reliably.